jina/executors/indexers/cache.py

```diff
@@ -62,15 +62,12 @@ def __init__(self, index_filename: Optional[str] = None, field: Optional[str] =
         if self.field not in self.supported_fields:
             raise ValueError(f"Field '{self.field}' not in supported list of {self.supported_fields}")
 
-    def add(self, doc_id: str, *args, **kwargs):
-        """Add a document to the cache depending on `self.field`.
+    def add(self, doc_id: str, data: str, *args, **kwargs):
```

Comment:
I don't think we should break the API of the inherited method.

Reply:
The inherited method from the `BaseKVIndexer` is:
`def add(self, keys: Iterable[str], values: Iterable[bytes], *args, **kwargs):`

Reply:
should we have the same interface?

Reply:
As discussed with @lusloher, adapting the interface should be done.